test/lit.cfg

```diff
@@ -905,6 +909,21 @@ elif run_os in ['linux-gnu', 'linux-gnueabihf', 'freebsd', 'windows-cygnus', 'wi
       config.target_sdk_name = "linux"
     config.target_runtime = "native"
     config.target_swift_autolink_extract = inferSwiftBinary("swift-autolink-extract")
+
+    libdispatch_build_dir = make_path(config.swift_obj_root, os.pardir,
+        'libdispatch%s' % config.variant_suffix, 'src')
+    libdispatch_build_artifacts = [
+        make_path(libdispatch_build_dir, 'libdispatch.so'),
+        make_path(libdispatch_build_dir, 'libswiftDispatch.so'),
+        make_path(libdispatch_build_dir, 'swift', 'Dispatch.swiftmodule')]
+    if (all(os.path.exists(p) for p in libdispatch_build_artifacts)):
+        config.available_features.add('libdispatch')
+        config.libdispatch_build_dir = libdispatch_build_dir
+        source_dir = make_path(config.swift_src_root, os.pardir, 'swift-corelibs-libdispatch')
+        swift_module_dir = make_path(libdispatch_build_dir, 'swift')
+        config.import_libdispatch = ('-I %s -I %s -L %s'
+            % (source_dir, swift_module_dir, libdispatch_build_dir))
```

Comment:
Nitpick: since Python local variables aren't scoped, can you stick `libdispatch_` on the front of `source_dir` and `swift_module_dir` too?

Reply:
I will do that. My logic was: it's inside the conditional (which is about libdispatch) so I thought it's redundant.

Reply:
Not in Python. :-(